readme.md

```diff
@@ -86,6 +104,7 @@ Pokémon names are available for the following languages:
 - `zh-Hans` (Simplified Chinese)
 - `zh-Hant` (Traditional Chinese)
 
+The language codes follow the [IETF BCP 47 standard.](https://en.wikipedia.org/wiki/IETF_language_tag)
```

Comment:
The previous link was to ISO 639-1, which is not actually the format used, it is BCP 47. The -Hans/-Hant is part of the IETF standard and not the ISO standard.